src/commandDetails/miscellaneous/uptime.ts

```diff
@@ -0,0 +1,41 @@
+import { container } from '@sapphire/framework';
+import {
+  CodeyCommandDetails,
+  SapphireMessageExecuteType,
+  SapphireMessageResponse,
+} from '../../codeyCommand';
+
+const uptimeExecuteCommand: SapphireMessageExecuteType = (
+  _client,
```

Comment:
Since the `client` parameter is being used, it shouldn't be prepended with an underscore.